lib/src/model/tournament/tournament_repository.dart

```diff
@@ -61,17 +61,28 @@ class TournamentRepository {
     );
   }
 
-  Future<Tournament> reload(Tournament tournament, {required int standingsPage}) {
+  Future<Tournament> reload(Tournament tournament) {
     return client.readJson(
       Uri(
         path: tournament.reloadEndpoint ?? '/api/tournament/${tournament.id}',
-        queryParameters: {'page': standingsPage.toString(), 'partial': 'true'},
+        queryParameters: {
+          if (tournament.standing != null) 'page': tournament.standing!.page.toString(),
+          'partial': 'true',
+        },
       ),
       headers: {'Accept': 'application/json'},
       mapper: (Map<String, dynamic> json) => tournament.updateFromPartialServerJson(json),
     );
   }
 
+  Future<Tournament> goToPage(Tournament tournament, int page) {
```

Comment:
`goToPage` is a bit weird in that context. Perhaps `loadPage`?

Reply:
yeah makes sense, it the end we *do* go to that page, but in the context of the repository `load` is more appropriate, agreed